app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, :post
+
```

Comment:
### Optional
- [ ] It would be better if we call the update method every time a comment is created and saved to the database and we can achieve that with the Rails Active Callbacks `after_save` method. So it would be like `after_save: update_comment_counter`

Reply:
Great idea, thank you for teaching me this syntax, I already applied this advice 👍